source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -197,28 +199,18 @@ the event of ties. For example, suppose we encounter a tie when applying a sort
 to the :ref:`sample collection <sorts-crud-sort-sample>` using the following
 code:
  
-.. io-code-block::
-
-   .. input:: /examples/generated/SortTest.snippet.handle-ties-1.kt
-      :language: kotlin
-
-   .. output:: 
-      :language:  console
-
-      FoodOrder(id=3, letter=a, food=maple syrup)
-      FoodOrder(id=5, letter=a, food=milk and cookies)
-      FoodOrder(id=2, letter=a, food=donuts and coffee)
+.. literalinclude:: /examples/generated/SortTest.snippet.handle-ties-1.kt
+   :language: kotlin
 
-Since multiple documents that matched our query contain the value "a" for the
-field on which we perform the sort, the documents could be returned
-in any order.
+Since multiple documents that matched the query contain the same value
+in the ``date`` field, the documents could be returned in a standard order.
```

Comment:
I'm not quite sure what "standard order" is referring to here? Is this a term we defined elsewhere or does it mean something like "the order the documents appear in the database"?

Reply:
oops, I think the wording was a little off here. I changed it to say
```
Since multiple documents that matched the query contain the same value
in the ``date`` field, the documents may not be returned in a consistent order.
```